runTests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+
+#export SYMFONY_VERSION="~2.7.0"
+SUPPORTED_SYMFONY_VERSIONS=('~2.3.0' '~2.5.0' '~2.6.0' '~2.7.0' '2.8.x-dev')
+export DOCTRINE_VERSION=">=2.2.3,<2.5-dev"
+
+sudo apt-get -qq update
```

Comment:
The installs should be removed as well.

I don't think that this script should check that the packages are installed. The contributing documentation should mention them and composer could check for their presence (it already does it for php5-mongo for instance).